### DISCUSSION TITLE
Enterprise and github copilot<title>

### DISCUSSION THREAD

**Discussion by ioVulture** [community/community#40539] [GitHub Education]
### Select Topic Area

Question

### Body

I have applied and been approved as a university under the assumption that copilot will be free for my staff (administrative, not only teachers or students). But when they try to activate it, it still charges them. I have connected my organisation which they are a member of. Can i assign copilot licenses in any way?

**Comment by ossanna16** [comment]
Hi there @ioVulture and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
